cloud/linode/linode.py

```diff
@@ -37,12 +37,95 @@
      - To keep sanity on the Linode Web Console, name is prepended with LinodeID_
     default: null
     type: string
+  displaygroup:
+    description:
+     - Add the instance to a Display Group in Linode Manager
+    default: null
+    type: string
+    version_added: "2.2"
```

Comment:
Ansible 2.2 has been branched already so this and the other instances for parameters you've added need chanming to `2.3`.